src/core-sdk-tasks/GenerateMSBuildExtensionsSWR.cs

```diff
@@ -24,22 +24,25 @@ public override bool Execute()
 
             AddFolder(sb,
                       @"MSBuildSdkResolver",
-                      @"MSBuild\Current\Bin\SdkResolvers\Microsoft.DotNet.MSBuildSdkResolver");
+                      @"MSBuild\Current\Bin\SdkResolvers\Microsoft.DotNet.MSBuildSdkResolver",
+                      ngenAssemblies: true);
 
             AddFolder(sb,
                       @"msbuildExtensions",
-                      @"MSBuild");
+                      @"MSBuild",
+                      ngenAssemblies: false);
 
             AddFolder(sb,
                       @"msbuildExtensions-ver",
-                      @"MSBuild\Current");
+                      @"MSBuild\Current",
+                      ngenAssemblies: false);
 
             File.WriteAllText(OutputFile, sb.ToString());
 
             return true;
         }
 
-        private void AddFolder(StringBuilder sb, string relativeSourcePath, string swrInstallDir)
+        private void AddFolder(StringBuilder sb, string relativeSourcePath, string swrInstallDir, bool ngenAssemblies)
```

Comment:
Suggestion, consider setting a default value to `false`, then you can omit the two calls above from passing the named parameter.

Reply:
Done, thank you.